source/aws_profile.c

```diff
@@ -592,12 +594,11 @@ static void s_profile_hash_table_value_destroy(void *value) {
  */
 
 void aws_profile_collection_destroy(struct aws_profile_collection *profile_collection) {
-    if (profile_collection == NULL) {
-        return;
-    }
+    aws_profile_collection_release(profile_collection);
+}
 
+static void aws_profile_collection_destroy_internal(struct aws_profile_collection *profile_collection) {
```

Comment:
trivial: we named the static functions to start with `s_*`
```suggestion
static void s_profile_collection_destroy_internal(struct aws_profile_collection *profile_collection) {
```

Reply:
Sorry, I have updated it.